src/encoder.rs

```diff
@@ -344,6 +346,53 @@ impl<T: Pixel> FrameState<T> {
     }
   }
 
+  pub fn compute_activity_mask(&mut self) {
+    //activity mask is computed only for luma plane
+
+    let luma_plane = &self.input.planes[0];
+    let PlaneConfig { width, height, .. } = luma_plane.cfg;
+    let luma = &luma_plane.data;
+    let PlaneConfig { xdec, ydec, .. } = self.activity_mask.cfg;
+    let mut act_mask = vec![0 as u32; (width >> xdec) * (height >> ydec) ];
```

Comment:
I mentally added the follwing to clarify that this is intended as an 8x8 mask:
```rust
assert!(xdec == 3);
assert!(ydec == 3);
```

Reply:
It is intended as an 8x8 mask. But if required, it could easily be changed to other block sizes by varying xdec and ydec in the PlaneCfg of the activity mask. It infers xdec and ydec from the `activity_mask` and computes accordingly.
Do you suggest that I add these assertions to be sure we're computing for 8x8 blocks?